### ISSUE TITLE
Pyupdate won't install (0.88)

### ISSUE THREAD

**Issue by filikun** [custom-components/custom_updater#99]
**Home assistant version: 0.88**

## Version of the custom_component
4.0.0

## Describe the bug
When trying to install the compontent I get a the error below. 

I've read some of the other issues that have been fixed here and tried the following:

DNS - Disabled Pi-Hole, still not working.
config\deps\lib - removed folders.
restarts - Tried restarting a couple of times. 
pycache- Removed and restarted. 

**I've seen you stated that one could install Pyupdate manually? How do I do that?**

## Configuration:
`custom_updater:`

## Log:
```
2019-02-25 10:28:40 WARNING (MainThread) [homeassistant.loader] You are using a custom component for custom_updater which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you do experience issues with Home Assistant.
2019-02-25 10:29:39 ERROR (SyncWorker_12) [homeassistant.util.package] Unable to install package pyupdate==1.3.1: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x75873a70>: Failed to establish a new connection: [Errno -3] Try again')': /simple/pyupdate/
  Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x758739b0>: Failed to establish a new connection: [Errno -3] Try again')': /simple/pyupdate/
  Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x75873a10>: Failed to establish a new connection: [Errno -3] Try again')': /simple/pyupdate/
  Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x75873950>: Failed to establish a new connection: [Errno -3] Try again')': /simple/pyupdate/
  Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x75873770>: Failed to establish a new connection: [Errno -3] Try again')': /simple/pyupdate/
  Could not find a version that satisfies the requirement pyupdate==1.3.1 (from versions: )
No matching distribution found for pyupdate==1.3.1
You are using pip version 19.0.2, however version 19.0.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
2019-02-25 10:29:39 ERROR (MainThread) [homeassistant.requirements] Not initializing custom_updater because could not install requirement pyupdate==1.3.1
2019-02-25 10:29:39 ERROR (MainThread) [homeassistant.setup] Setup failed for custom_updater: Could not install all requirements.
2019-02-25 10:29:44 ERROR (MainThread) [homeassistant.components.automation] Error while executing automation automation.restart_notification. Unknown error for call_service at pos 1: 
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/homeassistant/components/automation/__init__.py", line 375, in action
    await script_obj.async_run(variables, context)
  File "/usr/local/lib/python3.7/site-packages/homeassistant/helpers/script.py", line 131, in async_run
    await self._handle_action(action, variables, context)
  File "/usr/local/lib/python3.7/site-packages/homeassistant/helpers/script.py", line 210, in _handle_action
    action, variables, context)
  File "/usr/local/lib/python3.7/site-packages/homeassistant/helpers/script.py", line 299, in _async_call_service
    context=context
  File "/usr/local/lib/python3.7/site-packages/homeassistant/helpers/service.py", line 85, in async_call_from_config
    domain, service_name, service_data, blocking=blocking, context=context)
  File "/usr/local/lib/python3.7/site-packages/homeassistant/core.py", line 1133, in async_call
    self._execute_service(handler, service_call))
  File "/usr/local/lib/python3.7/site-packages/homeassistant/core.py", line 1155, in _execute_service
    await handler.func(service_call)
  File "/usr/local/lib/python3.7/site-packages/homeassistant/components/notify/__init__.py", line 117, in async_notify_message
    await notify_service.async_send_message(**kwargs)
  File "/usr/local/lib/python3.7/site-packages/homeassistant/components/notify/discord.py", line 43, in async_send_message
    import discord
  File "/usr/local/lib/python3.7/site-packages/discord/__init__.py", line 20, in <module>
    from .client import Client, AppInfo, ChannelPermissions
  File "/usr/local/lib/python3.7/site-packages/discord/client.py", line 38, in <module>
    from .state import ConnectionState
  File "/usr/local/lib/python3.7/site-packages/discord/state.py", line 36, in <module>
    from . import utils, compat
  File "/usr/local/lib/python3.7/site-packages/discord/compat.py", line 32
    create_task = asyncio.async
                              ^
SyntaxError: invalid syntax
```

**Comment by ludeeus** [comment]
Like I told you in Discord, this is a network error, if the network on the host works, try again later

**Comment by filikun** [comment]
> Like I told you in Discord, this is a network error, if the network on the host works, try again later

Sorry, did not see your reply! Will try again later then.

**Comment by ludeeus** [comment]
How did it go?

**Comment by filikun** [comment]
Haven't got it to work. Still the same issue 🙁

**Comment by ludeeus** [comment]
if you have pihole on the same device make sure you have set static ip (not reservation) and use a static upstream DNS like 1.1.1.1

**Comment by filikun** [comment]
> if you have pihole on the same device make sure you have set static ip (not reservation) and use a static upstream DNS like 1.1.1.1

So I did some tweaking. I changed the DNS on the router so it pointed to 1.1.1.1 instead of the Pi-Hole, I stopped the pi-hole addon on Hassio and deleted pycache and the old log and rebooted. Still get the same issue. I'm not sure how I do change the static IP (I have a reservation though). Is there a simple way of doing it? 

I have the Hassio on a Rpi 3b+.
